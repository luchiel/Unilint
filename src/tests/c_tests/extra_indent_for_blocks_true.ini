[spaces_and_newlines]
extra_indent_for_blocks = true