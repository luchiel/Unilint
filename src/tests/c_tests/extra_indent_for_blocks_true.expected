11:1: indent too small
12:1: indent too small
19:1: indent too small
20:1: indent too small
21:1: indent too small
30:1: indent too small
